func_env.sh

```diff
@@ -42,9 +42,8 @@ load_env() {
     esac
 
     sed -E 's/=(.*)$/= #\1/' .env.example >"$SERVICES_DIR/$ENV_FILE"
-    echo
-    echo "Please fill the $ENV_FILE file"
-    exit 1
+    
+    exec_in_place "$SERVICES_DIR/$ENV_FILE"
```

Comment:
I can't recall why you removed the echo and exit 1 here. Shouldn't it still show the message?

Reply:
if i remember correctly, the problem was the lines containing "$(cmd)" not being executed, which the "exec_in_place" command should fix now. if there's still something to be done with the file manually, i will readd the "exit" command after the new function